content/webinars/verify-your-infrastructure-on-deployment/index.md

```diff
@@ -0,0 +1,83 @@
+---
+# Name of the webinar.
+title: "Verify Your Infrastructure on Each Deployment with Pulumi and CircleCI"
+meta_desc: "Angel Rivera and Chris Smith will walk you through best practices for app developers, DevOps, and SREs to keep configuration mistakes from reaching production."
```

Comment:
This is sort of a meta-point, but IMO these should probably all be written in the present tense (even when they're in the future); it reads clearly enough that way, and then you don't have to go back and change all the tenses later when the day finally comes.